content/TrueCommand/SystemManagement/SingleSystem/TrueCommandSnapshots.md

```diff
@@ -1,42 +1,90 @@
+To see a list of snapshots for any other dataset, click on the dataset, then on **Snapshots**.
 
-To create single snapshots, select a pool in the system's *EXPLORE* menu and click *CREATE SNAPSHOTS*, then select *Create One-Time Snapshot*.
+![ExploreSystemDatasetSnapshotList](/images/TrueCommand/2.3.2/ExploreSystemDatasetSnapshotList.png "Dataset Snapshot list")
 
-![TrueCommandCreateOneTimeSnapshot](/images/TrueCommand/2.0/TC20TrueCommandCreateOneTimeSnapshot.png "Create One-time Snapshot")
+The header displays the breadcrumb for the system, pool, and dataset. To return to the pool, click the down arrow beside the system name, then select the pool on the dropdown list.
 
-Name the snapshot and click *CONFIRM*.
+## Taking Snapshots
+If the dataset does not have existing snapshots, a blue **CREATE SNAPSHOTS** button displays instead of a list of snapshots. 
 
-## Create Recurring Snapshot Tasks
+![CreateSnapshotButtons](/images/TrueCommand/2.3.2/CreateSnapshotButtons.png "Create Snapshot Buttons")
 
-To create recurring snapshot tasks, select a pool in the system's *EXPLORE* menu and click *CREATE SNAPSHOTS*, then select *Create Snapshot Task*.
+You can use this or the white **CREATE SNAPSHOTS** button under the *pool/dataset* path on the left-side panel. 
+**CREATE SNAPSHOTS** displays two options: 
 
-![TrueCommandCreateSnapshotTask](/images/TrueCommand/2.0/TC20TrueCommandCreateSnapshotTask.png "Create Snapshot Task")
+* **Create One-time Snapshot** to take a single snapshot
+* **Create Snapshot Task** to create a snapshot task to run on a schedule
 
-Set the task's schedule and determine the snapshot lifetime, then click *CONFIRM*.
+### Creating a Single Snapshot
+
+To create single snapshots:
+
+Click **EXPLORE** on the system card found on the dashboard, then select a pool on that system.
+
+Next click on a dataset you want to snapshot, then click **CREATE SNAPSHOTS**.
+
+![CreateSnapshotsOptions](/images/TrueCommand/2.3.2/CreateSnapshotsOptions.png "Create One-time Snapshot")
+
+Select **Create One-Time Snapshot** to open the snapshot dialog. 
+
+![CreateSnapshotDialog](/images/TrueCommand/2.3.2/CreateSnapshotDialog.png "Create Snapshot Dialog")
+
+Enter a name for the snapshot. 
+Select **Recursive** if the dataset selected has other datasets nested under it and you want to include those in the snapshot.
+
+Click **OK**. 
+
+## Creating a Snapshot Task
+
+To create snapshot tasks that occurs on a schedule:
+
+Click **EXPLORE** on the system card found on the dashboard, then select a pool on that system.
+
+Next click on a dataset you want to snapshot, then click **CREATE SNAPSHOTS**.
+
+![CreateSnapshotsOptions](/images/TrueCommand/2.3.2/CreateSnapshotsOptions.png "Create Snapshot Task")
+
+Select **Create Snapshot Task** to open the **Create Snapshot Task** configuration screen.
+If you did not select a dataset in the pool, you can select it from the dropdown list in the **Dataset** section, otherwise this area is not editable.

```

Comment:
```suggestion
Select **Create Snapshot Task** to open the **Create Snapshot Task** configuration screen.
If you did not select a dataset in the pool, you can select one from the dropdown list in the **Dataset** section. Otherwise, this area is not editable.

```